app_create.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import os
+from datetime import timedelta
+from pathlib import Path
+
+from azure.communication.email import EmailClient
+from dotenv import load_dotenv
+from quart import Quart
+from quart_cors import cors
+from quart_session import Session
+
+from emails import EmailConfig
+from environment_validation import validate_environment_settings
+from jwt_utils import JWTConfig
+
+
+def create_app():
+    load_dotenv()
+
+    jwt_config = JWTConfig(
+        client_public_keys_folder=Path(
+            os.getenv("MEMBRANE_CLIENT_PUBLIC_KEYS_DIRECTORY")
+        ),
+        server_public_key=Path(os.getenv("MEMBRANE_SERVER_PUBLIC_KEY")),
```

Comment:
I'm surprised the linter wouldn't enforce spaces before and after = sign. has this passed ruff linter check? if yes, we probably want to review the rules to enforce consistent spacing around operators

Reply:
Usually for method arguments, there is no space. Ruff forces this. I can look for a way to disable this.

Reply:
oh I see. No, It's fine. I guess the problem is we're doing in code what should be declarative which we'll fix later with env.yml.